webapps/api/backend/views/storagerouters.py

```diff
@@ -382,7 +387,7 @@ def update_framework(self, storagerouter):
     @log()
     @required_roles(['read', 'write', 'manage'])
     @return_task()
-    @load(StorageRouter)
+    @load(StorageRouter, max_version=6)
```

Comment:
I'd assume that you need to adapt the code below to call ```update_all``` with the right component?